### ISSUE TITLE
Hover and focus state for privacy policy not consistent with other form elements

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#222]
Hover and focus should be consistent. Current focus indicator also fails color contrast.
<img width="976" alt="Screenshot 2024-12-03 at 19 26 33" src="https://github.com/user-attachments/assets/aed1ba8e-0a35-45de-bf9a-66bd00a281cf">
